src/anurag/Bank3.java

```diff
@@ -0,0 +1,75 @@
+package anurag;
+
+public class Bank3 {
+
+	int debitamount;
+	int totalamount;
+	int currentamount;
+	int creditamount;
+	int counter1;
```

Comment:
what is meaning of counter1, counter2 and counter3
Give meaningful variable name.